### ISSUE TITLE
Spec needs additional info in the picklist

### ISSUE THREAD

**Issue by jvine** [sul-dlss/sul-requests#495]
Need to add the following to the picklists:
- [x] Comment field (not the "add volumes" comment, which is already there - the "I want to bring my dog" comment) (**request_comment**)
- [x] Planned date of use (**needed_date** I think)
- [x] Copy note (the public note that displays below the item in the Availability block and in the Request form) example: https://searchworks.stanford.edu/view/2148836

**Comment by jkeck** [comment]
We can add this content to the request to symphony at any point.  They can make use of that information at their convenience.

**Comment by ndushay** [comment]
Breadcrumbs:

https://github.com/sul-dlss/sul-requests/blob/master/app/jobs/submit_symphony_request_job.rb#L39-L43

**Comment by ndushay** [comment]
@jvine is it possible the "planned date of use"  is also the  "not needed after"  date?   I think we may use the same name for the field in the code (needed_date) but in the symphony request it becomes "not_needed_after":

```ruby
not_needed_after: (request.needed_date.strftime('%m/%d/%Y') if request.needed_date)
```

**Comment by ndushay** [comment]
also ... it seems we send in a single symphony request for multiple barcodes.   The barcodes are concatenated into a single value with `^` as a separator.   The public note you are asking for is associated with a particular barcode.   We would have to rejigger things (send in "item1" "item2" etc.) or do something clunky (join public note values with `^` separator and have empty notes when there are none?)  to send in info associated with individual barcodes ...

Maybe someone else has a better idea?

**Comment by ndushay** [comment]
for the copy note:  prefix copy note with `[barcode]: `;  separate copy notes with something (caret `^`) -- `copy_note`

for planned date of use:  we're already passing it as "not needed after" ... so Darsi will put it in pick list for appropriate  - leave it

for request_comment:  send as separate field;   `req_comment`

for existing item comments:  currently `comments`, change to `item_comments`

**Comment by ndushay** [comment]
Be sure to coord with Darsi when this is deployed so she has the code in place to deal with the new fields, esp. the rename to `item_comments`

**Comment by ndushay** [comment]
per planning meeting, we will deploy git branch `495-add-info-to-picklist`  to deploy target `requests-stage`  so @dlrueda can work on her end at her convenience.  `public_notes` implementation will follow with separate PR later.

**Comment by ndushay** [comment]
`495-add-info-to-picklist` is deployed to `requests-stage`

#634 will add the remaining bit, `copy_note` to the requests sent to symphony.

Once #634  is merged into `495-add-info-to-picklist` branch, and @dlrueda is ready, the deployment will go like this:

1.  Darsi will deploy her new code to prod.  It accepts `comments` and `item_comments` both, so it will work with current production and with these changes.
2.  after 1, we may deploy this fix to production.

**Comment by saseestone** [comment]
Tim Noakes just asked for the "requested on" date to also print on the picklist.  Thoughts?

**Comment by ndushay** [comment]
> Tim Noakes just asked for the "requested on" date to also print on the picklist. Thoughts?

@jvine  ^^

**Comment by ndushay** [comment]
We will add this scope creep, and call the param 

- [ ] `requested_date` -- for the "requested on" date

**Comment by dlrueda** [comment]
Work complete in Symphony code for -stage.
Submitted new picklists to Sarah Seestone for review. Expect review to be complete by today's retro.

Would like to coordinate moving this code to Symphony production with deploy to production requests.

**Comment by jvine** [comment]
Hey, Darsi - can I see the new picklists too? Is it easy to share them? I don’t think I’ve ever seen them, and I’m curious. (But if it’s not easy to share, I’ll continue to live.)

JV

On Nov 10, 2016, at 11:23 AM, Darsi Rueda <notifications@github.com<mailto:notifications@github.com>> wrote:


Work complete in Symphony code for -stage.
Submitted new picklists to Sarah Seestone for review. Expect review to be complete by today's retro.

Would like to coordinate moving this code to Symphony production with deploy to production requests.

**Comment by ndushay** [comment]
I would also like to see a picklist!  It would be helpful for understanding the gestalt of this whole thing.

**Comment by saseestone** [comment]
Jennifer I just sent you a copy.  Will send one to Naomi next :)
